testing/tests/matches.rs

```diff
@@ -26,6 +26,24 @@ fn test_match_option() {
     assert_eq!(s.render().unwrap(), "\nNot Found\n");
 }
 
+#[derive(Template)]
+#[template(path = "match-opt-bool.html")]
+struct MatchOptBoolTemplate {
+    item: Option<bool>
```

Comment:
Could you please add a comma to the end of the line to make "cargo fmt" happy? You can test it the linter wit `cargo fmt --all -- --check`.

Reply:
Done